src/tests/suite/daos_rebuild_simple.c

```diff
@@ -110,12 +116,18 @@ rebuild_akeys(void **state)
 			     data, DATA_SIZE, &req);
 
 	}
+
 	get_killing_rank_by_oid(arg, oid, 1, 0, &kill_rank, &kill_rank_nr);
 	ioreq_fini(&req);
 
 	rebuild_single_pool_target(arg, kill_rank, tgt, false);
+	rc = daos_obj_verify(arg->coh, oid, DAOS_EPOCH_MAX);
+	assert_true(rc == 0 || rc == -DER_NOSYS);
```

Comment:
maybe something like

if (rc != 0)
     assert_rc_equal(rc, -DER_NOSYS)

This would give us a message in jenkins output like

-DER_NOMEM != -DER_NOSYS

rather than 

rc == 0 || rc == -DER_NOSYS failed

Reply:
ah, sure.